DscResources/MSFT_EnvironmentResource/MSFT_EnvironmentResource.psm1

```diff
@@ -19,6 +25,10 @@ $script:maxUserEnvVariableLength = 255
         Retrieves the state of the environment variable. If both Machine and Process Target are
         specified, only the machine value will be returned.
 
+    .DESCRIPTION
```

Comment:
Resource description should not be added here since it was already added at the top